test/test_quantized_conv.py

```diff
@@ -8,45 +8,48 @@
 import torch.nn.quantized.functional as qF
 from torch.nn.quantized.modules.conv import _conv_output_shape
 
-from hypothesis import assume, given
+import hypothesis as hy
```

Comment:
I searched a bit, and the pattern people use is for hypothesis, we can do
`from hypothesis import xx as xxx` and for hypothesis_utils etc. we can do
`import hypothesis_utils as hu`